flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ShowFunctionsOperation.java

```diff
@@ -105,6 +107,14 @@ protected Collection<String> retrieveDataForTableResult(Context ctx) {
         }
     }
 
+    public String getDatabaseName() {
```

Comment:
related to my comment above, why all these getters if we can just make the fields protected. they are not used outside of the class hierarchy right?